fleetctl installs (#45552)
fleetdm/fleet: articles/fleetctl.md

## Installing fleetctl

> The easiest way to install fleetctl is from the [fleetdm.com/download page](https://fleetdm.com/download). To update fleetctl, re-install it.

Author: > To update fleetctl, re-install it.

@sgress454 is this right?

Reviewer: Yeah but might make more sense to just say, "the easiest way to install or update fleetctl is..." I know what you mean but when I see "re-install" I think of double-clicking the same installer I already downloaded.

Author: Good call 👍

> The easiest way to install fleetctl is from the [fleetdm.com/download page](https://fleetdm.com/download). To update fleetctl, re-install it.

Author: ```suggestion
The easiest way to install or update fleetctl is from the [fleetdm.com/download page](https://fleetdm.com/download).
```
